bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/messages.properties

```diff
@@ -1150,3 +1150,9 @@ OpenDelayedUrlAction_title=Open URL
 editorAssociationOverride_error_couldNotCreate_message=The ''{0}'' extension from plug-in ''{1}'' to the ''org.eclipse.ui.ide.editorAssociationOverride'' extension point failed to load the editor association override class.
 editorAssociationOverride_error_invalidElementName_message=An extension from plug-in ''{0}'' to the ''org.eclipse.ui.ide.editorAssociationOverride'' extension point was ignored because it contains the following invalid element: ''{1}''.
 editorAssociationOverride_error_invalidExtension_message=The ''{0}'' extension from plug-in ''{1}'' to the ''org.eclipse.ui.ide.editorAssociationOverride'' extension point will be ignored because it contains invalid attributes.
+
+User=User: {0}\n
+Host=Host: {0}\n
+Display=Display: {0}\n
+Process_Id=Process ID: {0}\n
+Workspace_Lock_Owner=Workspace lock is held by the following owner:\n{0}
```

Comment:
```suggestion
Workspace_Lock_Owner=Workspace lock is currently held by:\n{0}
```

Reply:
Please also check comment from Christoph above, I agree with him.

Reply:
Ok. I will change `Retry` button to `Cancel`. `Which effectively exits the launch`.
I will also interchange the buttons. Instead of `Retry` `Choose`. I will make them as `Choose` `Cancel`. Normally cancel always present at the end in all the eclipse dialogs.

Reply:
> Ok. I will change `Retry` button to `Cancel`. `Which effectively exits the launch`. I will also interchange the buttons. Instead of `Retry` `Choose`. I will make them as `Choose` `Cancel`. Normally cancel always present at the end in all the eclipse dialogs.

Please don't do that! Whatever **other** features are requested should be done in **separate** PR's.

Reply:
I've meant comment about change in the text to "Workspace lock is currently held by:\n{0}"